website/docs/docs/dbt-versions/release-notes.md

```diff
@@ -20,8 +20,24 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 
 ## June 2024
 
-- **New:** [Job warnings](/docs/deploy/job-notifications) are now GA. Previously, you could receive email or Slack alerts about your jobs when they succeeded, failed, or were canceled. Now with the new **Warns** option, you can also receive alerts when jobs have encountered warnings from tests or source freshness checks during their run. This gives you more flexibility on _when_ to be notified. 
+#### dbt Semantic Layer
+- **New:** Introduced a new granularity option for cumulative metrics, `period_agg`, nested under the `type_params` parameter. It specifies how to roll up the cumulative metric to another granularity. Options are `start`, `end`, `average`. Defaults to `start` if no `window` is specified.
+- **New:** Added validation that derived metrics must always have an `expr` key.
+- **New:** Enabled cache to pickup where filters that are included in saved queries.
```

Comment:
hey @Jstein77 and @plypaul - wanted to double check if this entry was the same as the july 8th feature where you can add filters to your query and it'll hit the cache? if so, i'm not sure if we should add it to this release note? see: https://linear.app/dbt-labs/issue/SL-2294/enable-cache-to-pickup-where-filters-that-are-included-in-saved#comment-a98da897